sdk/python/lib/test/test_next_serialize.py

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
```

Comment:
I added tests to this file as this is where some similar/related tests already were -- though, these new tests (and some of the existing ones) would prob make more sense in a new `test_output.py`.